OpenOversight/app/commands.py

```diff
@@ -65,11 +65,16 @@ def make_admin_user():
         username=username,
         email=email,
         password=password,
-        confirmed=True,
         is_administrator=True,
     )
     db.session.add(u)
     db.session.commit()
+
+    user = User.by_username(username).one_or_none()
```

Comment:
If you use `db.session.flush()` here,  I don't think you'llt need to look up the user again

Reply:
I'll give it a shot!